notebooks-html/bias_random_vars.html

```diff
@@ -314,8 +322,8 @@ <h3 id="Variance">Variance<a class="anchor-link" href="#Variance">&#182;</a></h3
 ></div></div></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<p>$ X $ takes on values -1 and 1 with probability $ \frac{1}{2} $ each. $ Y $ takes on values -2, -1, 1, and 2 with probability $ \frac{1}{4} $ each. We find that $ \mathbb{E}[X] = \mathbb{E}[Y] = 0 $. However, $ Y $'s distribution has a higher spread than $ X $'s and so we expect that $ Var(Y) $ is larger than $ Var(X) $.</p>
-$$
+<p>$ X $ takes on values -1 and 1 with probability $ \frac{1}{2} $ each. $ Y $ takes on values -2, -1, 1, and 2 with probability $ \frac{1}{4} $ each. We find that $ \mathbb{E}[X] = \mathbb{E}[Y] = 0 $. However, $ Y $'s distribution has a higher spread than $ X $'s, and so we expect that $ Var(Y) $ is larger than $ Var(X) $.</p>
```

Comment:
The last sentence feels a bit choppy. What about:

> Since $ Y $'s distribution has a higher spread than $ X $'s, we expect that $ Var(Y) $ is larger than $ Var(X) $.

Reply:
Changed.